### PR TITLE
MCKIN-26123 Platform Issues - Translations.

### PR DESCRIPTION
**Pull request by wasifarbisoft**

### CODE REVIEW

**Comment by wasifarbisoft** [comment]
@xitij2000 ,
Please review this PR for the translation.
Parallel PR for the python 2.7
https://github.com/open-craft/problem-builder/pull/313

File: problem_builder/translations/ar/LC_MESSAGES/text.po [hunk 1]
```diff
@@ -1498,15 +1498,15 @@ msgid ""
...
+"          You answered %(correct_answers)s questions correctly.\n"
 "        "
 msgstr[0] ""
 "\n"
 "          جاوبت عن سؤال واحد بشكل صحيح.\n"
 "        "
 msgstr[1] ""
 "\n"
-"          جاوبت عن (correct_answers)% من الأسئلة بشكل صحيح.\n"
+"          جاوبت عن %(correct_answers)s من الأسئلة بشكل صحيح.\n"
```
Comment (xitij2000) [inline problem_builder/translations/ar/LC_MESSAGES/text.po:1509]: This translation isn't showing up.

Comment (xitij2000) [reply problem_builder/translations/ar/LC_MESSAGES/text.po:1509]: Screenshot_2020-10-15 Unit Step Builder محتويات المساق OC101 Your Platform Name Here

Comment (wasifarbisoft) [reply problem_builder/translations/ar/LC_MESSAGES/text.po:1509]: Ah! Plural translations were missing. 
Fixed it!
thanks!

**Comment by wasifarbisoft** [comment]
@xitij2000 Fixed translations for plural strings
Please  review it again

**Comment by xitij2000** [comment]
@wasifarbisoft  Could you update the description to have more details for people who don't have access to the MCKIN ticket?

**Review by xitij2000 — COMMENTED** [review]
:+1:
- [x] I tested this: tested on master devstack
- [x] I read through the code

**Comment by wasifarbisoft** [comment]
@xitij2000 added the description to BOTH pull requests

**Comment by xitij2000** [comment]
@wasifarbisoft  Thanks!

**Pull request merged**